README.md

```diff
@@ -0,0 +1,12 @@
+# Archived Repository
+
+## Reason for Archiving
+
+This repository has been archived because the VScript has been implemented directly in the Counter-Strike: Source update dated 18.02.2025.
+
+## New Version
+
+The new version of the map now contains the VScript integrated directly into the BSP file.
+
+You can download the latest version here:
+[ze_jjba_v5fsv.bsp.bz2](https://fastdl.nide.gg/css_ze/maps/ze_jjba_v5fsv.bsp.bz2)
```

Comment:
dont put our links here since srcdslab is distinct with nide

Reply:
No, this is why I put nide fastdl link.

Reply:
well then dont put anything, i dont want people to ddos our fastdl